src/js/contexts/AnnounceContext/doc.js

```diff
@@ -14,14 +14,19 @@ export const doc = AnnounceContext => {
   DocumentedAnnounceContext.propTypes = {
     children: PropTypes.func.description(
       `Render function that will be called with an 'announce' function that
-      can be called when something should be announced. 
+      can be called when something should be announced.
       'announce' function accepts 'message', 'mode' and 'timeout' as arguments
       and these arguments can be passed as 'props' to the return component.
-      'mode' can be one of 'polite', 'assertive' or 'off'. 
+      'mode' can be one of 'polite', 'assertive' or 'off'.
       'timeout' is measured in milliseconds.
-      Example:  
-      {announce => 
-        <Button onClick={() => announce("Button was clicked", "polite", 1000)
+      Default is 'grommet-announcer'
```

Comment:
I don't think the context of this line makes sense since it's not connecting to the 'id' value. 
Regardless, I'm not sure it's relevant now that `id` isn't a prop.

Reply:
I agree with shimi on this on. id isn't a prop, so I don't think it needs to be in the docs here. If it is important, maybe we could think about accessibility documentation for grommet in the future. Besides that, pretty simple changes LGTM.

Reply:
@IanKBovard can you push a fix to this PR to fix the docs, it will speed up the reviewing.

Reply:
@ShimiSun @IanKBovard thanks for the discussion. I just pushed removing this line from the docs.